sensu-plugins-logs.gemspec

```diff
@@ -30,21 +30,22 @@ Gem::Specification.new do |s| # rubocop:disable Metrics/BlockLength
   s.test_files             = s.files.grep(%r{^(test|spec|features)/})
   s.version                = SensuPluginsLogs::Version::VER_STRING
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 2.7'
+  s.add_runtime_dependency 'sensu-plugin', '>= 3.0', '< 4.0'
```

Comment:
Thats fine but I'd prefer we keep the `~> 3.0` syntax to be consistent.

Reply:
that was the suggestion for from the dependbot pr. I was rolling it in.  Can we adjust the dependbot rules to make the prefered suggestion?

Reply:
fixed in latest commit

Reply:
Ya I asked them about it they can in a `Gemfile` but not currently in a `gemspec` though its possible its just not an easy change to make apparently. We can with some exceptions. Sometimes dependabot will allow multiple versions of a major version which makes it possible to unintentionally pull in breaking changes. I am fine with `'>= 3.0', < 4.0` but I am not fine with `>= 2.0', < 4.0`.